test.lua

```diff
@@ -4209,6 +4209,40 @@ function nntest.LookupTable()
    end
    local err = padw_sum - padw:sum()
    mytester:assertlt(err,precision, 'padding update error ')
+   -- test whether the weights changes accordingly when maxNorm is not nil
+   local all_index = torch.randperm(totalIndex):int()
+   -- input can have duplicates
+   local input = torch.repeatTensor(all_index:clone():narrow(1,1,nIndex), 2):int()
```

Comment:
I think this can just be:

`torch.repeatTensor(all_index:narrow(1,1,nIndex), 2)`